### PR TITLE
FIX: Catching product instance attributes from install request

### PR DESCRIPTION
**Pull request by jmmovilla**
***** REVIEWERS ****
@jframos

### CODE REVIEW

File: model/src/main/java/com/telefonica/euro_iaas/sdc/model/dto/ProductInstanceDto.java [hunk 1]
```diff
@@ -126,7 +109,7 @@ public String toString() {
         sb.append("[status = ").append(getStatus()).append("]");
         sb.append("[product = ").append(this.product).append("]");
         sb.append("[vm = ").append(getVm()).append("]");
-        sb.append("[attributes = ").append(this.attributes).append("]");
+        sb.append("[attributes = ").append(getAttributes()).append("]");
```
Comment (flopezag) [inline model/src/main/java/com/telefonica/euro_iaas/sdc/model/dto/ProductInstanceDto.java]: I do not understand, you delete the method getAttributes() but you add here the sentence with getAttributes(), it is correct?

Comment (jmmovilla) [inline model/src/main/java/com/telefonica/euro_iaas/sdc/model/dto/ProductInstanceDto.java]: Exactly. Attributes are already defined in InstallableInstance so there is no point to re-define them in ProductInstanceDto. This is the reason why attributes are not correctly taken in install request.

Comment (flopezag) [inline model/src/main/java/com/telefonica/euro_iaas/sdc/model/dto/ProductInstanceDto.java]: NTC

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**